src/root/components/RichTextOutput/index.tsx

```diff
@@ -10,18 +10,20 @@ function useSanitizedHtml(rawHtml: string) {
       rawHtml,
       {
         allowedTags: [
-          'b',
-          'i',
-          'em',
-          'h',
-          'p',
-          'bold',
-          'strong',
-          'li',
-          'ul',
-          'a',
-          'span',
-          'div',
+            // https://www.semrush.com/blog/html-tags-list
+            'p', 'br', 'hr', 'span', 'div',
+            'h1', 'h2', 'h3', 'h4', 'h5', 'h6',
+            'strong', 'b', 'i', 'em', 'u',
+            'li', 'ul', 'ol',
+            'a',
+            'table', 'thead', 'tbody', 'th', 'tr', 'td',
+            'dd', 'dt', 'dl',
+            'sub', 'sup',
+            'img', 'svg',
+            'pre', 'cite', 'code', 'q',
+            'base', 'iframe', 'canvas', 'video', // really needed?
+            'area', 'map', 'label', 'meter', // really needed??
```

Comment:
We should try to keep this one as minimum as possible. Also we need to be more careful about `iframe`

Reply:
Besides the "as minimum as possible" we should consider the "user perspective" also, which is mentioned here: https://github.com/IFRCGo/go-frontend/issues/2133#issuecomment-1184167219